genesyscloud/routing_queue/resource_genesyscloud_routing_queue_schema.go

```diff
@@ -33,6 +33,27 @@ var (
 		},
 	}
 
+	agentOwnedRoutingResource = &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"enable_agent_owned_callbacks": {
+				Description: "Enable Agent Owned Callbacks",
+				Type:        schema.TypeBool,
+				Required:    true,
+				Default:     false,
```

Comment:
I don't think it is necessary to have a default value if the field is required, since they will have to provide a value anyway. Also the default (or zero) value for all bool variables in golang is false anyway